jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -56,6 +55,8 @@ def buildExclusions = [
 
   // PPC BE ------------------------------------------------
   [ /^ppcbe-ubuntu/,                  anyType,     gte(8)  ],
+  [ /^ppcle-ubuntu/,                  anyType,     gte(13) ],
+  [ /^centos7-ppcle/,                 anyType,     lt(12)  ],
```

Comment:
These are le, not be (as the comment implies).

Reply:
Ack. I generalized the comment since IMO it more convenient to have these grouped.